src/domain/journey/subspace/routing/SubspaceRoute.tsx

```diff
@@ -27,13 +28,18 @@ const SubspaceRoute = () => {
         <Route path={SubspaceDialog.Index} element={<SubspaceHomePage dialog={SubspaceDialog.Index} />} />
         <Route path={SubspaceDialog.Subspaces} element={<SubspaceHomePage dialog={SubspaceDialog.Subspaces} />} />
         <Route path={SubspaceDialog.Contributors} element={<SubspaceHomePage dialog={SubspaceDialog.Contributors} />} />
+
         <Route path={SubspaceDialog.Activity} element={<SubspaceHomePage dialog={SubspaceDialog.Activity} />} />
         <Route path={SubspaceDialog.Timeline} element={<SubspaceHomePage dialog={SubspaceDialog.Timeline} />} />
         <Route path={SubspaceDialog.Share} element={<SubspaceHomePage dialog={SubspaceDialog.Share} />} />
         <Route
           path={`${SubspaceDialog.Timeline}/:${nameOfUrl.calendarEventNameId}`}
           element={<SubspaceHomePage dialog={SubspaceDialog.Timeline} />}
         />
+        <Route
+          path={SubspaceDialog.Settings}
+          element={journeyId && <Navigate to={`${buildJourneyAdminUrl(journeyId)}/profile`} />}
```

Comment:
don't use the `journeyId` here.
This is going to work, but it's going to generate a url like this:
![image](https://github.com/alkem-io/client-web/assets/16032487/e322dfff-df7d-4b29-8784-42920a33d442)
which works just because on the server it doesn't matter if we query `id` or `nameId`.

I'm taking a look myself, please don't merge this